### ISSUE TITLE
JAWS 2018/IE11: Accessible Name "Echo" for HTML Textbox with Nested Span in Explicit Label and aria-describedby attribute

### ISSUE THREAD

**Issue by jgauvreau** [FreedomScientific/VFO-standards-support#64]
### Summary

In IE 11 with JAWS 2018 (using default configuration settings), if you tab to a native HTML form control with an explicitly assigned HTML label, JAWS does not announce the correct accessible name for the control IF the control has both an explicitly assigned HTML label that includes a nested span element AND an aria-describedby attribute defined on the input tag.
[Real World Scenario: Design team wants to visually style the asterisk used as visible required field indicator in a different visual style from the rest of the label text and aria-describedby is used to include an accessible description of formatting guidance and/or an inline error message for the field.]

### Expected result

When Forms Mode is "on", expected JAWS to announce all the contents of the explicit label element (the text contents of the span followed by the rest of the text contents of the label) as the accessible name for the native HTML form control AND the calculated accessible description should be announced AFTER the accessible name, accessible role and any state/property information for the control.

### Actual result

JAWS has an "echo" when it announces the accessible name for the form control. See CodePen.io test page and notes/analysis in that page. 

In short, when tabbing and focus moved to the input textbox, JAWS 2018 announced:
* Organization Legal Name 3aOrganization Legal Name 3a Edit Required Use JawsKey+Alt+R to read descriptive text 
Enter legal business name filed with IRS
Type in text.

JAWS 2018 should NOT be making the second redundant announcement of "Organization Legal Name 3a". It seems like JAWS first announces the entire contents of the label element and then repeatedly announces the text node after the span element a second time

### CodePen.io Test Page with Examples

[https://s.codepen.io/jlgauvreau/debug/JLmpov](https://s.codepen.io/jlgauvreau/debug/JLmpov)

### Additional Information
- HTML code from CodePen.io test case was tested and validated against Nu HTML Checker
- aViewer shows the correct accessible name calculated for the input control (MSAA and UIA)
- aViewer shows an accessible description calculated for the input control in MSAA (not UIA) if a tabindex="-1" attribute is added to the <div> containing the formatting guidance/tip since it is the element whose id is referenced in the aria-describedby attribute. for the input. (This is a known/documented workaround for IE 11)

#### JAWS version and build number
JAWS 2018.1803.24

 
#### Operating System and version
Windows 7

#### Browser and version:
IE 11

**Comment by hanshillen** [comment]
Confirmed, filed internally as bug # 100679.

**Comment by jgauvreau** [comment]
Thanks @hanshillen !

**Comment by TPGJRogers** [comment]
Issue is marked IE11. Closing.
